### ISSUE TITLE
Add high scores option to SuperTux

### ISSUE THREAD

**Issue by tobbi** [SuperTux/supertux#482]
From Twitter: https://twitter.com/petittux/status/740671016356175872

I wonder whether we should add a high scores option to SuperTux. We could then allow people to enter their name etc. Might be fun if you're playing SuperTux with more than one player.

**Comment by brmbrmcar** [comment]
I think the profiles should also be done by custom name, with the ability to add more.

**Comment by tobbi** [comment]
This is not what this issue is about.

**Comment by brmbrmcar** [comment]
Right! I have had it!

**Comment by Karkus476** [comment]
@brmbrmcar Feel free to make a new issue, or comment on one which already addresses your issue. It's a good idea. 
@tobbi Do you mean high scores of coins, time & badguys etc. ?

**Comment by tobbi** [comment]
@Karkus476 You're right, maybe I should've thought about this some more. We need a way to compare high scores against each other. Or maybe only use time as an indicator.

@brmbrmcar For god's sake, stop acting like a child. I told you that "Your suggestion is not what this issue is about" -> "open a new issue, as Karkus suggested".

You cannot act pissy only because I tell you to follow processes, and I am honestly tired of your bitchy attitude. Oh, and if you want to make a video about this (where you do not react to my comments AT ALL just to show THE WHOLE WORLD how MEAN the SuperTux developers are, go right ahead, I don't f***ing care).

**Comment by Karkus476** [comment]
@tobbi Time as an indicator is okay, but not as impressive as assigning a name to your high score

**Comment by tobbi** [comment]
Yes, but how do we *rank* highscores in the first place...That's what I meant.

**Comment by Karkus476** [comment]
@tobbi I see... Maybe different 'tabs' could show the 'top' ten for each variable: lowest time, highest badguys, highest coins

**Comment by tobbi** [comment]
That could work, however, a lot of people will have the "highest badguys" and "highest coins", because those are finite, and there can be a best score.
